website/docs/plugin/framework/handling-data/terraform-concepts.mdx

```diff
@@ -0,0 +1,729 @@
+                            Optional: true,
+                        },
+                        "list_attribute": schema.ListAttribute{
+                            Optional:    true,
+                            ElementType: types.StringType,
+                        },
+                    },
+                },
+            },
+
+            "map_nested_attribute": schema.MapNestedAttribute{
+                Optional: true,
+                NestedObject: schema.NestedAttributeObject{
+                    Attributes: map[string]schema.Attribute{
+                        "map_attribute": schema.MapAttribute{
+                            Optional:    true,
+                            ElementType: types.StringType,
+                        },
+                        "number_attribute": schema.NumberAttribute{
+                            Optional: true,
+                        },
+                    },
+                },
+            },
+
+            "set_nested_attribute": schema.SetNestedAttribute{
+                Optional: true,
+                NestedObject: schema.NestedAttributeObject{
+                    Attributes: map[string]schema.Attribute{
+                        "object_attribute": schema.ObjectAttribute{
+                            Optional: true,
+                            AttributeTypes: map[string]attr.Type{
+                                "bool_attribute":    types.BoolType,
+                                "float64_attribute": types.Float64Type,
+                                "int64_attribute":   types.Int64Type,
+                                "list_attribute":    types.ListType{ElemType: types.StringType},
+                                "map_attribute":     types.MapType{ElemType: types.StringType},
+                                "number_attribute":  types.NumberType,
+                                "set_attribute":     types.ListType{ElemType: types.StringType},
+                                "string_attribute":  types.StringType,
+                            },
+                        },
+
+                        "set_attribute": schema.SetAttribute{
+                            Optional:    true,
+                            ElementType: types.StringType,
+                        },
+                        "string_attribute": schema.StringAttribute{
+                            Optional: true,
+                        },
+                    },
+                },
+            },
+
+            "single_nested_attribute": schema.SingleNestedAttribute{
+                Optional: true,
+                Attributes: map[string]schema.Attribute{
+                    "bool_attribute": schema.BoolAttribute{
+                        Optional: true,
+                    },
+                    "float64_attribute": schema.Float64Attribute{
+                        Optional: true,
+                    },
+                },
+            },
+        },
+    }
+}
+```
+
+## Blocks
```

Comment:
For a followup PR, I think we should just include a quick summary here similar to the top of the blocks page, then link to that page for the rest. 👍